docs/paper/dev/api/scheduler.md

```diff
@@ -0,0 +1,197 @@
+﻿---
+slug: /dev/scheduler
+---
+
+# Task Scheduler
```

Comment:
Why is the page called task scheduler if you immediately refer to it with the `BukkitScheduler` name. Maybe just name the page "The Bukkit Scheduler" or something along those lines?

Reply:
I'd prefer something like `Scheduling tasks` over including Bukkit in the name